Document/MediaManager.php

```diff
@@ -34,7 +34,7 @@ public function save($entity, $andFlush = true)
             $entity->setProviderName(func_get_arg(2));
         }
 
-        if ($andFlush && is_bool($andFlush)) {
+        if (is_bool($andFlush)) {
```

Comment:
The whole point of this commit is to get rid of comparing this to true as you might want to pass false value to the save method.